arroyo/processing/strategies/decoder/json.py

```diff
@@ -0,0 +1,26 @@
+from typing import Optional
+
+import fastjsonschema
```

Comment:
one thing I like to do is put a helper comment on the first import so ti shows in the stacktrace -- like `# pip install sentry-arroyo[json]`